nextcloudappstore/core/views.py

```diff
@@ -14,6 +15,9 @@ class AppDetailView(DetailView):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['categories'] = Category.objects.all()
+        context['latest_releases_by_platform_v'] = \
+            self.object.latest_releases_by_platform_v()
+        context['desc_markdown'] = markdown(self.object.description)
```

Comment:
While the code did not run, it did indeed become `<a href="javascript:alert(1)">link name</a>`.